js/bitget3.js

```diff
@@ -0,0 +1,2899 @@
+                        'order/history',
+                        'order/historyPlan', // conditional
+                        'trace/closeTrack',
+                        'trace/currentTrack',
+                        'trace/historyTrack',
+                        'trace/summary',
+                        'trace/profitSettleTokenIdGroup',
+                        'trace/profitDateGroupList',
+                        'trace/profitDateList',
+                        'trace/waitProfitDateList',
+                    ],
+                    'post': [
+                        'account/leverage',
+                        'account/adjustMargin',
+                        'account/modifyAutoAppendMargin',
+                        'order/placeOrder',
+                        'order/batchOrders',
+                        'order/cancel_order',
+                        'order/cancel_batch_orders',
+                        'order/plan_order',
+                        'order/cancel_plan',
+                        'position/changeHoldModel',
+                        'trace/closeTrackOrder',
+                    ],
+                },
+            },
+            'fees': {
+                'spot': {
+                    'taker': this.parseNumber ('0.001'), // Reduced 20% to 0.0008 if fee is paid with BGB instead
+                    'maker': this.parseNumber ('0.001'), // Reduced 20% to 0.0008 if fee is paid with BGB instead
+                },
+                'swap': {
+                    'taker': this.parseNumber ('0.0006'),
+                    'maker': this.parseNumber ('0.0004'),
+                },
+            },
+            'requiredCredentials': {
+                'apiKey': true,
+                'secret': true,
+                'password': true,
+            },
+            'exceptions': {
+                // http error codes
+                // 400 Bad Request — Invalid request format
+                // 401 Unauthorized — Invalid API Key
+                // 403 Forbidden — You do not have access to the requested resource
+                // 404 Not Found
+                // 500 Internal Server Error — We had a problem with our server
+                'exact': {
+                    '1': ExchangeError, // { "code": 1, "message": "System error" }
+                    // undocumented
+                    'failure to get a peer from the ring-balancer': ExchangeNotAvailable, // { "message": "failure to get a peer from the ring-balancer" }
+                    '4010': PermissionDenied, // { "code": 4010, "message": "For the security of your funds, withdrawals are not permitted within 24 hours after changing fund password  / mobile number / Google Authenticator settings " }
+                    // common
+                    // '0': ExchangeError, // 200 successful,when the order placement / cancellation / operation is successful
+                    '4001': ExchangeError, // no data received in 30s
```

Comment:
Suggestion: some new exceptions that can be added:

`{"code":"40017","msg":"Parameter verification failed,Number of ordersRange error","requestTime":1643901683652,"data":null}`
`{"code":"40017","msg":"Parameter verification failed,Number of ordersRange error","requestTime":1643900237504,"data":null}`

Reply:
Fixed in PR #11907